e2e/cypress/integration/ui/settings/03_team_add_users.spec.ts

```diff
@@ -8,20 +8,20 @@ describe('team add users', () => {
   const descriptionForTeam = cypressPrefix + ' team ' + now;
   const nameForTeam = 'testing-team-' + now;
   let teamID = '';
-  let adminToken = '';
+  let adminIdToken = '';
```

Comment:
renamed these variables to distinguish the admin user's ID token from the generated admin API token saved in the environment